nowcasting_dataset/data_sources/fake/batch.py

```diff
@@ -125,16 +135,26 @@ def gsp_fake(
     return GSP(xr_dataset)
 
 
-def metadata_fake(batch_size):
-    """Make a xr dataset"""
+def metadata_fake(batch_size, temporally_align_batches: int = False) -> Metadata:
```

Comment:
tiny typo, I think?  :slightly_smiling_face: 

```suggestion
def metadata_fake(batch_size, temporally_align_batches: bool = False) -> Metadata:
```